docs/upgrade-notes/migrate-tea-standalone.md

```diff
@@ -0,0 +1,148 @@
+---
+id: migrate_tea_standalone
+title: Migrate TEA deployment to standalone module
+hide_title: false
+---
+
+## Background
+
+Previous versions of Cumulus included deployment of the Thin Egress App (TEA) by default in the `distribution` module. As a result, this forced Cumulus users who wanted to deploy a new version of TEA to wait on a new release of Cumulus that incorporated that release.
+
+In order to give Cumulus users the flexibility to deploy newer versions of TEA whenever they want, deployment of TEA has been removed from the `distribution` module and **Cumulus users must now add the TEA module to their deployment**. [Guidance on integrating the TEA module to your deployment is provided](deployment/thin-egress-app.md), or you can refer to [Cumulus core example deployment code](https://github.com/nasa/cumulus/blob/master/example/cumulus-tf/main.tf).
```

Comment:
Nit: 

It might be good to reference the specific component in main.tf.

Reply:
Add TEA module resource name